src/applications/Applications.SmiRunner/ServiceVerbs.cs

```diff
@@ -15,6 +15,9 @@ public abstract class ApplicationVerbBase : VerbBase
         protected new const string BaseHelpText = VerbBase.BaseHelpText + "src/applications/Applications.";
     }
 
+    [Verb("dicom-loader",HelpText = "Load DICOM files")]
```

Comment:
Should we add "Into MongoDb" here to highlight its role in ETL?

Reply:
Done

Reply:
Done